lib/Constants.php

```diff
@@ -204,4 +221,8 @@ class Constants {
 	];
 
 	public const DEFAULT_FILE_FORMAT = 'csv';
+
+	public const UNSUBMITTED_FILES_FOLDER = self::FILES_FOLDER . '/unsubmitted';
+
+	public const FILES_FOLDER = 'forms';
```

Comment:
Now that I see it in action, I think we should use `Forms` with capital F :)

```suggestion
	public const FILES_FOLDER = 'Forms';
```

Reply:
done